gapic/src/main/com/google/gapid/LoadingScreen.java

```diff
@@ -46,12 +75,95 @@ public LoadingScreen(Composite parent, Theme theme) {
     titleLabel.setFont(theme.bigBoldFont());
     titleLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
 
+    Label versionLabel = createLabel(container, "Version " + GAPID_VERSION.toFriendlyString());
+    versionLabel.setForeground(theme.welcomeVersionColor());
+    versionLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
+
     statusLabel = createLabel(container, "Starting up...");
-    statusLabel.setLayoutData(new GridData(SWT.LEFT, SWT.TOP, true, false));
+    statusLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
+
+    GridLayout gridLayout = new GridLayout(3, false);
+    gridLayout.horizontalSpacing = 15;
+    optionsContainer = Widgets.createComposite(container, gridLayout);
+    optionsContainer.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
+    this.createOptions();
   }
 
   public void setText(String status) {
     statusLabel.setText(status);
     statusLabel.requestLayout();
   }
+
+  /**
+   * Hide the messaging box and display the links after server set up.
+   */
+  @SuppressWarnings("hiding")
+  public void showOptions(Client client, Models models, Widgets widgets) {
+    this.client = client;
+    this.models = models;
+    this.widgets = widgets;
+
+    statusLabel.setVisible(false);
+    optionsContainer.setVisible(true);
+    if (models.settings.getRecent().length <= 0) {
+      removeRecentOption();
+    }
+  }
+
+  /**
+   * Initialize the links for layout settings. Hide them until server set up.
+   */
+  private void createOptions() {
+    createLabel(optionsContainer, "", theme.add());
+    createLink(optionsContainer, "<a>Capture a new trace</a>", e -> {
+      showTracingDialog(checkNotNull(client), getShell(), checkNotNull(models), checkNotNull(widgets));
+    });
+    Label captureHint = createLabel(optionsContainer, (OS.isMac ? "\u2318" : "Ctrl") + " + T");
+    captureHint.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, false));
+    captureHint.setForeground(theme.welcomeVersionColor());
```

Comment:
Don't reuse the version color. Create a new color in the theme. It would surprise someone to see the hint color change if they want to change the version color.